_posts/2019-05-11-Neue_Testing.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: post
+title:  "neue Testing 2018.2.1+bremen1"
+author: genofire
+date:   2019-05-11 14:55:21 +0200
+---
+
+Es gibt eine neue testing-Firmware für die Knoten des Bremer Freifunks: 2018.2.1+bremen1
+
+Damit sind wir wieder auf den aktuellsten Stand der Gluon Entwicklung.
+Unter anderem können nun einige Geräte der Serie "Fritz!WLAN Repeater" verwendet werden.
+
+Ein bekanntes Problem ist weiterhin, dass in einigen WLAN-Chipsets der Betrieb des Mesh-Netzes und des Client-Netzes auf 5 GHz nicht funktioniert.
```

Comment:
Also soweit ich das erinnere, gibt es nur mit dem *Mesh*-Netzwerk Probleme (https://github.com/freifunk-gluon/gluon/issues/1584). Die 5GHz-Probleme mit dem *Client*-Netz sind in https://github.com/freifunk-gluon/gluon/issues/1561 gelöst worden!